pages/a/orchestrating-crate-with-docker-112.html

```diff
@@ -0,0 +1,148 @@
+title: Orchestrating Crate with Docker 1.12
+link: https://crate.io/blog/orchestrating-crate-with-docker-112
```

Comment:
Good article! A couple of general thoughts/comments:
#1 - I don't think the article made it clear what is done differently in the GA 1.12 release vs. the release candidate? What changed? As you go along, maybe you can point out what got changed/fixed/added in 1.12 GA?
#2 - This reads well as a piece of documentation for Crate users...but is there something we can say or change that makes this a learning for people who don't use Crate? Maybe change the title "How to orchestrate a SQL Database cluster with Docker 1.12" to make it more generic (and findable/SEO)? Add a little more wording up top that explains how Docker 1.12 makes it much easier to do this?
#3 - sort of related to point 2, and I'd only do this if it doesn't create a ton of new writing and reading here...but can you show how to resize a cluster w/ Docker? Normally a hard thing to do with a SQL DB? but not with Crate/Docker?

Danke Claus!